Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx

```diff
@@ -823,3 +824,68 @@ void vtkMRMLApplicationLogic::EditNode(vtkMRMLNode* node)
   // Observers in qSlicerCoreApplication listen for this event
   this->InvokeEvent(EditNodeEvent, node);
 }
+
+//----------------------------------------------------------------------------
+void vtkMRMLApplicationLogic::RegisterModuleLogic(const char* moduleName,
```

Comment:
We should be able to use `const std::string& moduleName` as it is now wrapper in python.

Reply:
It would be nice to harmonize this in Slicer overall. Until we have `const char*` at some places, it is a bit complicated to set a string because if we get a string from somewhere as const char* then we always need to check if it is nullptr and if it is then replace it with "" (because setting nullptr as std::string& input immediately crashes the application).